Cargo.toml

```
@@ -3,61 +3,54 @@
 # Copyright 2022 SECO Mind Srl
 #
 # SPDX-License-Identifier: CC0-1.0
+
+[workspace]
+members = [
+    "e2e-test",
+    "hardware-id-service",
+    "led-manager-service",
+]
+
 [package]
 name = "edgehog-device-runtime"
 version = "0.7.1"
 default-run = "edgehog-device-runtime"
 edition = "2021"
 homepage = "https://github.com/edgehog-device-manager/edgehog-device-runtime"
-rust-version = "1.59.0"
+rust-version = "1.66.1"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
```

Specifying a workspace dependency to be inherited by members of a workspace.
```suggestion
[workspace.dependencies]
```